service/host_test.go

```diff
@@ -195,7 +195,7 @@ func TestHostShutdown_RunningService(t *testing.T) {
 func TestHostShutdown_CloseSuccessful(t *testing.T) {
 	sh := makeRunningHost()
 	sh.serviceCore.metricsCore = metricsCore{
-		scope:            tally.NoopScope,
+		metrics:          tally.NoopScope,
```

Comment:
nit: not sure i like renaming metrics to scope. Yes, we expose this publicly through `Metrics()` API, but internally we don't need to mix and match things, i.e. `call it like you see it`. It's a tally scope.